### ISSUE TITLE
Migrate from MarkDownSharp to Markdig

### ISSUE THREAD

**Issue by FranklinYu** [chocolatey/chocolatey.org#683]
Would be a good idea to migrate to Markdig (or any other [CommonMark compliant implementation](https://github.com/commonmark/commonmark-spec/wiki/List-of-CommonMark-Implementations#c-1)). Even [MarkDownSharp](https://github.com/StackExchange/MarkdownSharp) itself recommends CommonMark.

**Comment by designxsteph** [comment]
@FranklinYu I have already completed this conversion and it will be released soon after the redesign!

**Comment by FranklinYu** [comment]
You mean it’s included in #653?

**Comment by designxsteph** [comment]
It's not included in #653, but is dependent on it. Once that is merged (soon), then I will put in the PR for this. Migrating to Markdig also has fixed issue #680.

**Comment by FranklinYu** [comment]
Awesome. Shall I close this issue now, or when those said edits get merged into master branch?

**Comment by designxsteph** [comment]
Let’s go ahead and keep it open for now and I’ll reference it in my commits. 👍🏻
